src/widgets/sparkline.rs

```diff
@@ -253,6 +253,7 @@ mod tests {
 
     #[test]
     fn it_does_not_panic_if_max_is_set_to_zero() {
+        #[expect(clippy::unnecessary_min_or_max)]
```

Comment:
This is a known clippy [bug](https://github.com/rust-lang/rust-clippy/issues/13191).  Rather than `ignore` it and link to the issue, I used the new `expect` so that it will fail as soon as the bug is resolved and the lint is no longer needed.

Reply:
Posted as you did I think.